src/s3/commands/presignedURL.ts

```diff
@@ -41,33 +47,115 @@ export async function presignedURLCommand(
     })
 }
 
-export async function promptTime(fileName: string, window = Window.vscode()): Promise<number> {
-    const timeStr = await window.showInputBox({
-        value: '15',
-        prompt: localize(
-            'AWS.s3.promptTime.prompt',
-            'Specify the time (minutes) until URL will expire for path: {0}',
```

Comment:
was this message dropped intentionally? It shows the path in the prompt:

![image](https://user-images.githubusercontent.com/55561878/219652387-a0a96cf3-3e4e-4f9b-9212-edfbe72dc9ad.png)